package.json

```diff
@@ -5,7 +5,7 @@
   "main": "dist/index.js",
   "scripts": {
     "ts-node": "ts-node",
-    "test:lint": "eslint './src/**/*.ts' './test/**/*.ts' --fix",
+    "test:lint": "eslint \"./src/**/*.ts\" \"./test/**/*.ts\" --fix",
```

Comment:
Guess that fixes issues with running linter checks on win-based machines. Thanks 👍